lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -175,8 +174,22 @@ export default class BitcoinProcessor {
       }
     }
 
-    console.info(`Returning transactions since ${since ? 'block ' + TransactionNumber.getBlockNumber(since) : 'begining'}...`);
-    let transactions = await this.transactionStore.getTransactionsLaterThan(since, this.pageSize);
+    // wait until at least one block is processed
```

Comment:
This check should not be required. The service doesn't start even listening until it has caught up to the latest block. See the ````processTransactions(...)```` call in the ````initialize()```` function. Also, the ion\bitcoin.ts where the initialize function is called.

Reply:
Good point.